### ISSUE TITLE
Add on-prem option in cli

### ISSUE THREAD

**Issue by vanneback** [elastisys/ck8s-cluster#3]
A few things that need to be done for this task:
* Add option in cli
* "Split up" apply step into terraform (infra) and ansible (install k8s) on the on-prem option.
* Make apply accept manually created ansible inventory file.

**Comment by simonklb** [comment]
I would like to reword this to say "accept manual cluster state" instead of a "manually created Ansible inventory file". I think the inventory file should be generated the same way as if you use Terraform output as the cluster state and not by the user.
